### PR TITLE
Fixed missing Internationalization and updated Chinese Simplified

### PR DESCRIPTION
**Pull request by geeksteph**
## Proposed Changes
1. 

## Additional Info.
Any additional info goes here

### CODE REVIEW

File: src/calendar/CalendarError.tsx [hunk 1]
```diff
@@ -107,6 +108,8 @@ const Map = styled.img`
   }
 `;
 
+const { t } = useTranslation("common");
```
Comment (wseagar) [inline src/calendar/CalendarError.tsx:111]: For this to work it needs to be within the CalendarError component

File: src/HomePage.tsx [hunk 2]
```diff
@@ -54,13 +55,16 @@ const StyledTabs = styled.div`
   }
 `;
 
+
+const { t } = useTranslation("common");
```
Comment (wseagar) [inline src/HomePage.tsx:59]:
```
const Tabs: FunctionComponent<TabsProps> = ({ activeTab }) => {
  const { t } = useTranslation("common");
  return (
  <StyledTabs role="tablist">
    <PageLink to="/bookings">
      <Tab isActive={activeTab === TabType.bookings}>Make a Booking</Tab>
      <Tab isActive={activeTab === TabType.bookings}>{t("core.makeABooking")}</Tab>
    </PageLink>
    <PageLink to="/locations">
      <Tab isActive={activeTab === TabType.walkIn}>Walk-in/Drive Thru</Tab>
      <Tab isActive={activeTab === TabType.walkIn}>{t("core.walkInDriveThru")}</Tab>
    </PageLink>
  </StyledTabs>
  )
};

**Comment by geeksteph** [comment]
This is now ready for review.

**Comment by geeksteph** [comment]
This is now ready for review.

**Pull request merged**